### DISCUSSION TITLE
[공통컴포넌트] Input

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#4] [General]
<img width="274" alt="스크린샷 2022-06-07 오후 4 33 58" src="https://user-images.githubusercontent.com/72402747/172322512-2ce3855d-6781-4a00-9f75-2de42af5c391.png">

**Comment by hyunjin0910** [comment]
## Props
* label
* error message
* value

onChange
input type validation
placeholder
